src/Google/Http/MediaFileUpload.php

```diff
@@ -263,7 +292,12 @@ public function getUploadType($meta)
     return self::UPLOAD_MULTIPART_TYPE;
   }
 
-  private function getResumeUri()
+  public function getResumeUri()
+  {
+    return ( $this->resumeUri !== null ? $this->resumeUri : $this->fetchResumeUri() );
```

Comment:
According to the README.md, PHP 5.2 is supported, but according to [the docs](http://php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary), this was introduced in 5.3?